api/serializers.py

```diff
@@ -235,7 +235,12 @@ def validate(self, data):
         return data
 
     def get_occurrences(self, event):
-        occurrences = EventOccurrence.objects.filter(event=event, start__gt=datetime.now())
+        request = self.context.get('request')
+        user = getattr(request, 'user', None)
+        if request and getattr(user, 'winery', None) and user.winery == event.winery:
+            occurrences = EventOccurrence.objects.filter(event=event)
+        else:
+            occurrences = EventOccurrence.objects.filter(event=event, start__gt=datetime.now())
```

Comment:
nit, we could rewrite this as:
ocurrences = ...
if user_is_not_the_owner()... :
    occurences = ocurrences.filter(start_gt=datetime.now())

Reply:
Done